### ISSUE TITLE
🛑 RustDesk Public Rendezvous Server is down

### ISSUE THREAD

**Issue by rustdesk** [rustdesk/uptime#8]
In [`189cd1e`](https://github.com/rustdesk/uptime/commit/189cd1ea72f13d4ff6887d496e4ff2a9d8c08f5b
), RustDesk Public Rendezvous Server (rs-ny.rustdesk.com) was **down**:
- HTTP code: 0
- Response time: 0 ms

**Comment by indig0F10w** [comment]
IS down.
![Capture](https://github.com/rustdesk/uptime/assets/17021548/8e04b406-e084-4e50-a670-44d019d3c10c)

**Comment by sck1979** [comment]
hi,
already down?
![Uploading errore-rustdesk.JPG…]()

**Comment by Swiftgti** [comment]
Is still down, not working RustDesk

**Comment by Tilomey** [comment]
It's not working for several days - since last week :(
Will it be back online at some point, or is it over?

**Comment by cyber-sec0** [comment]
still down

**Comment by ByPyodaLP** [comment]
is it down again can someone confirm

**Comment by indig0F10w** [comment]
It is not working. It was good until about hour or two ago.

**Comment by Tilomey** [comment]
sometimes it is online again, but it is not stable and goes off again after a few minutes

**Comment by terralogia** [comment]
I was working ok at last two hours but, it´s down again

**Comment by Hollul** [comment]
I think it's working again. I could connect to my remote computers :D

**Comment by rc621** [comment]
Thank you very much for the new connection.. it's going great !!

**Comment by brequete** [comment]
For me it's still down, have been like 2 weeks with this issue

![image](https://github.com/rustdesk/uptime/assets/35566109/ee7eb7f2-e553-4a9a-8737-43e389478ba1)

**Comment by cyber-sec0** [comment]
It worked for a few days until today, and since a few hours ago it got down again.

The good thing is that I was already connected, so the connection kept working.

The issue is the first time that is tries to connect, as long as that works once, it works forever until you disconnect.

@rustdesk 
There should be 1 extra rendezvous server (or 1 rendezvous server for each server location). This would solve that issue.

**Comment by rustdesk** [comment]
https://twitter.com/rustdesk/status/1755787729814573213

**Comment by cyber-sec0** [comment]
> https://twitter.com/rustdesk/status/1755787729814573213

Ok...

So in case I'm banned...
Where do I go to prove I am not DDOSing the Rust desk servers?

**Comment by cyber-sec0** [comment]
Well nvm, the server was just permanently deleted, banned and locked? Wow...

You didn't tell the server owners anything at all in a timely manner?

Any plans to get a working free public server somewhere else?

**Comment by sck1979** [comment]
Good mornig,
some days ago works fine , today don't work
![rustdesk-error-09-02-2024-h0915](https://github.com/rustdesk/uptime/assets/158469618/551da390-6e12-412b-8f8e-bb780ba5f03e)

**Comment by ninjaeon** [comment]
> Well nvm, the server was just permanently deleted, banned and locked? Wow...
> 
> You didn't tell the server owners anything at all in a timely manner?
> 
> Any plans to get a working free public server somewhere else?

There's several other threads on Rustdesk's Twitter feed that address your questions.

TLDR; They are still working with [Vultr](https://twitter.com/Vultr) to rectify this.

https://twitter.com/rustdesk
https://twitter.com/rustdesk/status/1753735166915194886
https://twitter.com/rustdesk/status/1754818959637942671
https://twitter.com/rustdesk/status/1754747491583479929
https://twitter.com/rustdesk/status/1753859553895051393

**Comment by rustdesk** [comment]
Thanks @ninjaeon

**Comment by rustdesk** [comment]
Up
